JSDemos/Demos/DataGrid/BatchUpdateRequest/description.md

```diff
@@ -0,0 +1,13 @@
+If data is stored on a server, the widget can save all changes with a single request. To add this functionality, set **editing**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode) to *"batch"* and implement the [onSaving](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onSaving) function. It accepts an `e` object that contains fields necessary for batch update. The following steps describe the function's implementation:
```

Comment:
I guess the question here is why doesn't the grid do this automatically? If I turn on batch updates, and then I press Commit/Submit/Save (whatever the name might be), why do I have to provide custom implementation? Why doesn't the grid save it all in a single request?

Reply:
The grid doesn't save data automatically in batch editing mode **when data is stored on a server only**. Do you mean that "If data is stored on a server when **editing**.[mode]() is *"batch"*, implement the [onSaving]() function. This will allow users to save all changes with a single request." is better?

Reply:
If editing mode is batch, by default the widget sends >= 1 requests when a user saves data. This is because there are individual insert/update/remove operations and the widget can't send them all in one request - by default.
To send only one request, a custom saving logic must be implemented and this demo shows how to do that.

Reply:
I like this version. The only thing I'd maybe change is more the information about batch update settings to the top (all of this relates to batch updates only, right?). So maybe start like this:

If **editing**.[mode](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/editing/#mode) is *"batch"*, end-users can modify multiple records and submit all changes at once. If data is stored on a server ...